admin_manual/configuration_files/encryption_configuration.rst

```diff
@@ -3,30 +3,29 @@ Encryption Configuration
 ========================
 
 If you are upgrading from ownCloud 8.0, and have encryption enabled, please see 
-:ref:`upgrading_encryption_label` (below) for the correct steps to upgrade your encryption. 
+:ref:`upgrading_encryption_label` (below) for the correct steps to upgrade your 
+encryption.
 
-In ownCloud 8.1 and up the server-side encryption has a number of changes and 
-improvements, including:
+The primary purpose of the ownCloud server-side encryption is to protect users' 
+files on remote storage, such as Dropbox and Google Drive, and to do it easily 
+and seamlessly from within ownCloud.
 
-* When encryption is enabled, all files are no longer encrypted at user's first 
-  logins because this causes timeouts on large installations. Instead, only 
-  files that are created or updated after encryption has been enabled are 
-  encrypted.
+In ownCloud 8.2 the server-side encryption has a number of changes and 
+improvements, including:
 
-* The "decrypt all" option in the Personal settings has been removed, also for 
-  performance reasons.
-
-* A new option for users to enable/disable encryption on a per mount-point 
-  basis.
-  
-* The option to choose from multiple encryption modules.
+* An option to create a master encryption key, which replaces all individual 
+  user keys. This is especially useful for single-sign on.
+* Encrypt all data files at once when enabling encryption.
+* Decrypt all data files, or per user.
+* Users may decrypt their own files.
+* Migrate all keys after a major upgrade, or per user.
```

Comment:
this was already possible for oc8 -> oc8.1 migration. From oc8.1 to oc8.2 no migration steps are necessary